### PR TITLE
Add/media import improvements

### PR DESCRIPTION
**Pull request by ariskataoka**
## Description

This PR uses the failure details URL for the import media command. 

## Pull request checklist

- [ ] Update [SETUP.md](https://github.com/Automattic/vip-cli/blob/trunk/docs/SETUP.md#list-of-environmental-variables) with any new environmental variables.
- [ ] Update [the documentation](https://github.com/Automattic/vip-cli/blob/trunk/docs).
- [ ] [Manually test](https://github.com/Automattic/vip-cli/blob/trunk/docs/TESTING.md#manual-testing) the relevant changes.
- [ ] Follow the [pull request checklist](https://github.com/Automattic/vip-cli/blob/trunk/docs/RELEASING.md#new-pull-requests)
- [ ] Add/update [automated tests](https://github.com/Automattic/vip-cli/blob/trunk/docs/TESTING.md#automated-testing) as needed.

## New release checklist

- [ ] [Automated tests](https://github.com/Automattic/vip-cli/blob/trunk/docs/TESTING.md#automated-testing) pass.
- [ ] The [Preparing for release checklist](https://github.com/Automattic/vip-cli/blob/trunk/docs/RELEASING.md#preparing-for-release) is completed.

## Changelog Description

Media import: Add option to download failure details logs from URL returned in the media import request response body. 

## Steps to Test

**Obs.:** This test requires a locall VIP API  public up and running. 

1. Check out PR.
1. Run `npm run build`

### Scenario 1:

1. Import media files by running `  VIP_PROXY="" API_HOST={ LOCAL_VIP_API_PUBLIC_URL } node ./dist/bin/vip-import-media.js @{ TEST_SITE_ID }.{ ENV_SITE } { MEDIA_TO_BE_IMPORTED_URL }`

The output should contain a message about the errors and a prompt as:
```
⚠️  Error details can be found on ${ ERROR_DETAILS_URL_RETURNED_BY_VIP_PUBLIC_API }
Download error details now? (y/N) › 
```

1. Answer y
2. The output should contain information about the download and the local file as a result:

```
⚠️  All errors have been exported to { LOCAL_PATH }
```

### Scenario 2:

Repeat step 1 
1. Answer n
2. The prompt should exit

### CODE REVIEW

File: src/lib/media-import/status.ts [hunk 1]
```diff
@@ -271,44 +273,123 @@ ${ maybeExitPrompt }
...
+			progressTracker.suffix += `${ chalk.red(
+				`Could not export errors to file\n${ ( writeFileErr as Error ).message }`
+			) }`;
+		}
+	}
+
+	async function fetchFailureDetails( fileErrorsUrl: string ) {
+		progressTracker.suffix += `
+=============================================================
+Downloading errors details from ${ fileErrorsUrl }...
```
Comment (t-wright) [inline src/lib/media-import/status.ts]:
```suggestion
Downloading errors details from ${ fileErrorsUrl } ...
```

Currently when clicking the URL from a terminal, the ellipsis is treated as part of the URL which breaks the presigned URL

File: package.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@automattic/vip",
-  "version": "2.39.7-dev.0",
+  "version": "2.40.0-dev.2",
```
Comment (sjinks) [inline package.json:3]: Please don't do this. The [prepare release workflow](https://github.com/Automattic/vip-cli/actions/workflows/npm-prepare-release.yml) updates the package version.

File: npm-shrinkwrap.json [hunk 3]
```diff
@@ -1,12 +1,12 @@
 {
   "name": "@automattic/vip",
-  "version": "2.39.7-dev.0",
+  "version": "2.40.0-dev.2",
```
Comment (sjinks) [inline npm-shrinkwrap.json:3]: Same as above

File: npm-shrinkwrap.json [hunk 4]
```diff
@@ -1,12 +1,12 @@
...
   "lockfileVersion": 2,
   "requires": true,
   "packages": {
     "": {
       "name": "@automattic/vip",
-      "version": "2.39.7-dev.0",
+      "version": "2.40.0-dev.2",
```
Comment (sjinks) [inline npm-shrinkwrap.json:9]: Same as above

**Review by sjinks — CHANGES_REQUESTED** [review]

**Pull request merged**